File: DataModel/Models/CalculoHistoricoModel.cs
using DataModel.Models.Enum;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DataModel.Models
{
    public class CalculoHistoricoModel : BaseModel
    {
        public decimal? ValorPrimario { get; set; }
        public Operacoes? Operacao { get; set; }
        public decimal? ValorSecundario { get; set; }
        public decimal? Resultado { get; set; }
        public string Usuario { get; set; }
        public string Calculo { get; set; }

    }
}

File: CalcSka/ClientApp/src/models/calculoHistoricoModel.ts
import BaseModel from "./baseModel";
import Operacoes from "./enum/Operacoes";

export default class CalculoHistoricoModel extends BaseModel {
  valorPrimario: number = null;
  operacao: Operacoes = null;
  operacaoString: string = null;
  valorSecundario: number = null;
  resultado?: number = null;
  usuario: string = null;
  calculo: string = null;
};
